packages/SwingSet/docs/vat-upgrade.md

```diff
@@ -0,0 +1,308 @@
+(NOTE: none of this is implemented yet)
+
+# Vat Upgrade
+
+Dynamic vats can be upgraded to use a new code bundle. This might be used to
```

Comment:
Yeah, the authority model is way less clear for static vats. The bootstrap vat doesn't seem right: it's special, and controls all access to the static vats, but doesn't already have the power to modify their internals.

It probably wants to be a method on the controller. I'll leave it out of this doc because I don't know what to even speculate about.